chain/cmd/provider/main_test.go

```diff
@@ -0,0 +1,48 @@
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"testing"
+
+	"github.com/bandprotocol/d3n/chain/x/zoracle"
+	"github.com/stretchr/testify/require"
+	abci "github.com/tendermint/tendermint/abci/types"
+	common "github.com/tendermint/tendermint/libs/common"
+	//sdk "github.com/cosmos/cosmos-sdk/types"
+	// "github.com/stretchr/testify/require"
+)
+
+func TestExecWithTimeoutSuccess(t *testing.T) {
```

Comment:
We will see if all these tests cause the CI to be too slow. We can decide how to deal with that later if it ever becomes a problem.